README.md

```diff
@@ -1,20 +1,52 @@
+Recreate the figures from https://www.biorxiv.org/content/10.1101/505032v3 in 4-ish easy steps!
 
+The DeepCell Kiosk paper [(Bannon et al., 2020)](<https://www.biorxiv.org/content/10.1101/505032v3>) presents cost and runtime benchmarks for running a generic image segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b).
+ 
 N.B.: Before following any of these steps, I'd recommend activating a Python `virtualenv`. It won't affect how easy it is to run this project, probably, but it will make your life easier after you've forgotten about this project and moved on to other Python packages.
 
 (To view API documentation for this project online, please go [here](https://deepcell-kiosk-figure-generation.readthedocs.io).)
 
 ## Figure Creation
 1) To create the figures, first install the `deepcell-kiosk-figure-generation` package using the included `setup.py` file with the command `pip install -e .`.
-2) Place all JSON benchmarking data in a top-level folder inside the project. (The project comes with a `new_data` folder containing default data for this purpose, but feel to throw this data away and replace it with your own, or put yours in an entirely new folder.)
+2) Place all JSON benchmarking data in a top-level folder inside the project. (The project comes with a `new_data` folder containing default data for this purpose, but feel to throw this data away and replace it with your own, or put yours in an entirely new folder.) (Also, see the section `Regenerating DeepCell Kiosk Paper Benchmarking Data` for help on recreating the figures from the DeepCell Kiosk paper.)
 3) Edit the global attributes in `figure_generation/graph_creation.py` to suit your tastes. (The global attributes should all be documented at the top of `figure_generation/graph_creation.py`.)
 4) Execute `python3 ./figure_generation/graph_creation.py`.
 
 Note that we did a significant amount of aesthetic primping in Illustrator to the figures generated from the included default data before we put them in the paper, but the data is all the same.
 
-## Documentation
+## Regenerating DeepCell Kiosk Paper Benchmarking Data
+
+This repository can produce the benchmarking figures from the paper, but it expects a full battery of benchmarking output files from all run conditions. It will not produce any figures as-is without output from all runs. 
+ 
+To generate a full battery of benchmarking run data (i.e., varying image numbers and GPU numbers), please complete the following benchmarking runs:
+ 
+     - 3 runs with 1 GPU and 10,000 images
+     - 3 runs with 4 GPUs and 10,000 images
+     - 3 runs with 8 GPUs and 10,000 images
+     - 3 runs with 1 GPU and 100,000 images
+     - 3 runs with 4 GPUs and 100,000 images
+     - 3 runs with 8 GPUs and 100,000 images
+     - 1 run with 1 GPU and 1,000,000 images
+     - 1 run with 4 GPUs and 1,000,000 images
+     - 1 run with 8 GPUs and 1,000,000 images
```

Comment:
The `version2` branch assumes that there are 3 runs for everything.

Reply:
Thanks for catching that! Fixed.